### PR TITLE
testing the tests

### PR DESCRIPTION
**Pull request by sopos**
an ability to run changed tests by a PR on fedora-rawhide and epel-all

### CODE REVIEW

File: changed-tests.fmf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+summary: check the changed tests functionality
+discovery:
```
Comment (psss) [inline changed-tests.fmf:2]:
```suggestion
discover:
```

A small typo...

**Comment by sopos** [comment]
here is the tracking issue of packit to allow tests only https://github.com/packit/packit-service/issues/1078

File: tests.spec [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Name:       tests
+Version:    1
+Release:    1
+Summary:    Dummy just to trigger tests
+License:    MIT
+BuildArch:  noarch
```
Comment (lachmanfrantisek) [inline tests.spec:6]: Looks like we can't work without a source to be specified. I've created an issue for that: https://github.com/packit/packit/issues/1251

The source will be replaced by the archive of this repository (by default) so the content of that variable should not be something meaningful.

File: .packit.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+specfile_path: tests.spec
+
+upstream_package_name: tests
+downstream_package_name: tests
+
```
Comment (lachmanfrantisek) [inline .packit.yaml:5]: Maybe an easier option using [actions](https://packit.dev/docs/actions/) to overwrite the default workflow:


```suggestion

actions:
  fix-spec-file: "true"

```

File: tests.spec (hunk 2 above)
Comment (lachmanfrantisek) [reply tests.spec:6]: There is a workaround for that -- see the other comment.

Comment (sopos) [reply tests.spec:6]: Thanks for helping me out but the problem is still there.
```
Message: Preparation of the repository for creation of an SRPM failed: min() arg is an empty sequence
Exception: PackitSRPMException('Preparation of the repository for creation of an SRPM failed: min() arg is an empty sequence')
```
https://dashboard.packit.dev/results/srpm-builds/24382

Comment (lachmanfrantisek) [reply tests.spec:6]: OK, there is another place we work with sources. To overcome this, you can add a `sources` directory. Its existence will turn off downloading of the sources.

With that change, I've been able to build `SRPM` via packit locally.

Comment (TomasTomecek) [reply tests.spec:6]: I can see tests to be running \o/ glad you guys solved this out

Comment (sopos) [reply tests.spec:6]: Yep, thank you for you assistance.

There's still some issue as it reports it is still running even if the test is already finished.

**Comment by lachmanfrantisek** [comment]
It's weird that it ended with `needs-inspection` result that was unknown to us and caused a parsing error (fixed in https://github.com/packit/packit-service/pull/1099). 

The results page of TF can be found as `Web URL` on our dashboard, e.g.:
* https://dashboard.packit.dev/results/testing-farm/24720 -> http://artifacts.dev.testing-farm.io/ed5fa4ee-8264-45f6-adcf-8fde9ca2f9c5

File: tests.spec (hunk 2 above)
Comment (TomasTomecek) [reply tests.spec:6]: that will be fixed on monday

**Comment by sopos** [comment]
Looks like it finally propagates the correct results. Thanks for you assistance.

**Comment by TomasTomecek** [comment]
> Looks like it finally propagates the correct results. Thanks for you assistance.

I skimmed through the logs and it seems all tests passed but here I can see that 'Tests failed' - is that really expected?

**Comment by sopos** [comment]
> I skimmed through the logs and it seems all tests passed but here I can see that 'Tests failed' - is that really expected?

You're right. I can see PASS for Fedora run and fail for CentOS runs. So there may be some other issue.

**Comment by psss** [comment]
The [/usbguard/Sanity/config-sanity/rule_options](http://artifacts.dev.testing-farm.io/19a32df7-024d-4bce-aae6-abebccc77afc/work-changed-testst2GIJA/tmt-run.log) ended with a `warn`. Not sure why the result is not reported. @thrix?

**Comment by sopos** [comment]
refer to https://github.com/RedHat-SP-Security/tests/pull/36 to see the tests behavior

**Pull request merged**